pyRevitPlus.tab/Smart Align.panel/smartalign.stack3/Horizontal.pulldown/Align Center.pushbutton/script.py

```diff
@@ -23,8 +23,8 @@
 
 if not VERBOSE:
   #__window__.Close()
-
-ALIGN = Align.HCENTER
+    
+    ALIGN = Align.HCENTER

```

Comment:
All these should not be indented. We can remove. The if not verbose

Reply:
For line 26 you are correct.

On my computer when I remove the indentation at line 27 and reload Pyrevit - the Reload window has an Error for unexpected indeted block. 

ERROR: [pyrevit.extensions.genericcomps] Error parsing script file: C:/Users/_______/pyRevitPlus.tab/Smart Align.panel/smartalign.stack3/Horizontal.pulldown/Align Center.pushbutton/script.py | Error parsing script file: C:/Users/_______/pyRevitPlus.tab/Smart Align.panel/smartalign.stack3/Horizontal.pulldown/Align Center.pushbutton/script.py | expected an indented block 

Traceback: 
 File "C:/Users/_______/pyrevitlib/pyrevit/extensions/genericcomps.py", line 322, in _analyse_python_script 
 script_content = ScriptFileParser(self.get_full_script_address()) 

Thoughts?